### DISCUSSION TITLE
My project does not render apparently due to unknown width of uploaded image file

### DISCUSSION THREAD

**Discussion by VolkerJacobsen** [neuefische/questions#637] [Web]
Hello everybody, 

Unfortunately I am stuck with my ProjectFrom. Although the formdata is commited successfully the new project does not render due to unknown image width. 

Error Message: UNHANDELED RUNTIME ERROR / Error: Image with src "/assets/image/data:image/jpeg;base64,/..." is missing required "width" property. 

[US2]https://github.com/VolkerJacobsen/capstone-volker/tree/us2-form-submit

**Comment by actyvystom** [comment]
Please provide correct code snippets and a link to your affected project/branch so that we can get more context to figure out the problem.
Thanks, Thomas

**Comment by VolkerJacobsen** [comment]
Hi Thomas, thanks for your feedback. I now linked the branch. Apparently the code snippet is too long to be posted. The respective code is shown in the ProjectForm.js

**Comment by julialoeschel** [comment]
Hey. Your Problem migth be in index.js
The Image Component requires the following properties: src, width, height, and alt.
please check the [docs here](https://nextjs.org/docs/pages/api-reference/components/image)
